kitsune/sumo/templatetags/jinja_helpers.py

```diff
@@ -211,6 +211,14 @@ def datetimeformat(context, value, format='shortdatetime'):
     Returns a formatted date/time using Babel's locale settings. Uses the
     timezone from settings.py, if the user has not been authenticated.
     """
+
+    def get_user_timezone(user):
```

Comment:
If not split into another inner function, the code looks messy and the code become hard to read. I think, getting the User timezone should be in another function.